bloom-housing/bloom
# chore: add translations for the Confirmation form step

## Issue Overview

This PR addresses #2734 

- [x] This change is a dependency for another issue: [#393](https://github.com/housingbayarea/bloom/issues/393)

## Description

This PR includes the core translations for the content on the Confirmation application form step. Note that it doesn't include the specific "Prepare for Next Steps" copy which will be addressed downstream (see https://github.com/housingbayarea/bloom/issues/395)

## How Can This Be Tested/Reviewed?

Apply and fill out the application, verify the copy on the confirmation screen in various languages.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
